docs_src/microservices/device/details/DeviceDiscovery.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Device Service - Device Discovery and Provision Watchers
+---
+
+# Device Service - Device Discovery and Provision Watchers
+
+Device Services may contain logic to automatically provision new devices.  This can be done **statically** or **dynamically**.  
+
+### Static Provisioning
+
+In static device configuration (also known as static provisioning) the device service connects to and establishes a new device that it manages in EdgeX (specifically metadata) from configuration the device service is provided.  For example, a device service may be provided with the specific IP address and additional device details for a device (or devices) that it is to onboard at startup.  In static provisioning, it is assumed that the device will be there and that it will be available at the address or place specified through configuration.  The devices and the connection information for those devices is known at the point that the device service starts.
```

Comment:
```suggestion
In static provisioning, the device service is provided with a Device file which contains the definition(s) of the device(s) to statically provision. The device service connects to and establishes a new device that it manages in EdgeX (specifically metadata) from device definition configuration the device service is provided.  For example, a device service may be provided with the specific IP address and additional device details for a device (or devices) that it is to onboard at startup.  In static provisioning, it is assumed that the device will be there and that it will be available at the address or place specified through the device definition configuration.  The devices and the connection information for those devices is known at the point that the device service starts.
```